src/imperator/data/graphical_culture_types.rs

```diff
@@ -20,8 +19,7 @@ impl GraphicalCultureType {
 }
 
 impl DbKind for GraphicalCultureType {
-    fn validate(&self, key: &Token, block: &Block, data: &Everything) {
-        let mut vd = Validator::new(block, data);
+    fn validate(&self, _key: &Token, _block: &Block, _data: &Everything) {
         // Literally nothing
```

Comment:
In this case it's good to have a separate
`Validator::new(block, data);`
that you don't assign to anything, in order to validate that the block indeed contains nothing. The Validator will do that check when it is dropped.